modules/social_features/social_group/social_group.services.yml

```diff
@@ -1,4 +1,8 @@
 services:
+  plugin.manager.join:
```

Comment:
Most of the services are based on the "default_plugin_manager" service and located in Drupal core or contrib modules using the same format so do we really need to use a custom format?

Reply:
Hi @OH17L, great observation. 

I would like then to suggest the following:  `plugin.manager.social_group.join`, the usage of the module namespace is very important to avoid issues with duplicated names. This choice is sustained by the following examples:

`migrate.services.yml` -> `plugin.manager.migrate.process`, `plugin.manager.migrate.source`
`media.services.yml` -> `plugin.manager.media.source`
`views.services.yml`-> `plugin.manager.views.access`, `plugin.manager.views.area`

and other services declared by Drupal Core itself.

Please let me know your thoughts!

Reply:
Thank you